### PR TITLE
Migrate to latest Storybook

### LINKED ISSUE #397: Migrate to newest version of Storybook
# Feature request
**Is your feature request related to a problem? Please describe.**
#247 has been open for a long time.

**Describe the solution you'd like**
Bump Storybook and relevant add-ons to latest version and resolve any bugs that show up in `yarn storybook` and `yarn build-storybook`. Change whatever is needed to get it to work again!

### PR DESCRIPTION
**Pull request by KrDimitrov**
# Description of changes
- Upgraded all packages to latest except `husky`, `lint-staged`, and `next-redux-wrapper`. 
- latest storybook working
- fixed `svg` importing being broken due to latest upgrades. 

# Issue Resolved

Fixes #397

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,27 +43,26 @@
...
-    "@storybook/addon-viewport": "3.4.11",
-    "@storybook/addons": "3.4.11",
-    "@storybook/react": "3.4.11",
+    "@storybook/addon-a11y": "5.0.11",
+    "@storybook/addon-actions": "5.0.11",
+    "@storybook/addon-backgrounds": "5.0.11",
+    "@storybook/addon-info": "5.0.11",
+    "@storybook/addon-knobs": "5.0.11",
+    "@storybook/addon-storyshots": "5.0.11",
+    "@storybook/addon-viewport": "5.0.11",
```
Comment (kylemh) [inline package.json]: Can you add carets before the version numbers for the Storybook dependencies?

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit 1acc972594a34484ecf901764c21f27e32c1d3e7

https://deploy-preview-508--operation-code-storybook.netlify.com

**Comment by kylemh** [comment]
@KrDimitrov I can't seem to edit anything on your fork.

Did you check off the box mentioned in [this blogpost](https://github.blog/2016-09-07-improving-collaboration-with-forks/)?

**Comment by KrDimitrov** [comment]
> 
> 
> @KrDimitrov I can't seem to edit anything on your fork.
> 
> Did you check off the box mentioned in [this blogpost](https://github.blog/2016-09-07-improving-collaboration-with-forks/)?

It is checked

File: package.json (hunk 1 above)
Comment (KrDimitrov) [reply package.json]: Done!

**Comment by kylemh** [comment]
I think everything we’ve done is correct. I’m left to assume that the alpha version of `@zeit/next-css` isn’t working properly, as we’re getting incorrect lexical errors from optimize-css-assets plugin. (edited) 

I think I’ll have to create my own plugin to ensure that Next can work with `css-loader@2.x`

I’m gonna do my best to make a custom storybook webpack config to let us use `css-loader@1.x` which should let us merge your Storybook update without issues.

Then… I guess we’ll write our own `next-css` and maybe contribute back to `next-plugins` repo. I wont do that until after we release the site.

All of this, unfortunately, is a lower priority for me right now, so we'll leave this PR as `On Hold`. You're free to take a whack at the above, of course.

**Comment by kylemh** [comment]
@KrDimitrov let me know on Slack if you're interested in paired programming to contribute to Zeit's next-plugin repo.

File: package.json [hunk 2]
```diff
@@ -43,18 +43,17 @@
...
+    "@storybook/addon-backgrounds": "^5.0.11",
+    "@storybook/addon-info": "^5.0.11",
+    "@storybook/addon-knobs": "^5.0.11",
+    "@storybook/addon-storyshots": "^5.0.11",
+    "@storybook/addon-viewport": "^5.0.11",
+    "@storybook/addons": "^5.0.11",
+    "@storybook/react": "^5.0.11",
     "@zeit/next-bundle-analyzer": "^0.1.2",
-    "@zeit/next-css": "^1.0.1",
+    "@zeit/next-css": "0.2.1-canary.4",
```
[the remark this answers is not in the document]
Comment (kylemh) [reply package.json]: Yeah, they’ve goofed up the versioning on that.

**Comment by kylemh** [comment]
@KrDimitrov I know it has been a hot minute, but I went ahead and did #877 and #880 today which made this PR very trivial since you've already done all the work 💃 

I did notice that there are a few broken stories, but we've finally made progress here!

**Comment by KrDimitrov** [comment]
Alright, ill see what I can do!

**Comment by kylemh** [comment]
I can finish this out if you're uninterested! I was just excited to let you know that _it's happening_! Let me know what you prefer.

I could have you make a new PR if you'd like to get some credit for Hacktoberfest?

**Comment by KrDimitrov** [comment]
> I could have you make a new PR if you'd like to get some credit for Hacktoberfest?

That would be great, thanks!

**Comment by KrDimitrov** [comment]
Also, I don't know about which branch to use, update_storybook is ahead for the migration, but there are a lot of conflicts with master. should I merge them?

**Comment by kylemh** [comment]
@KrDimitrov probably just close this, fork again, and just treat your fork's master as a new branch

**Pull request closed without merging**